Tests/SentryTests/Integrations/ViewHierarchy/SentryViewHierarchyIntegrationTests.swift

```diff
@@ -53,6 +53,12 @@ class SentryViewHierarchyIntegrationTests: XCTestCase {
         XCTAssertFalse(sentrycrash_hasSaveViewHierarchyCallback())
     }
 
+    func test_integrationAddFileName() {
+        SentrySDK.start { $0.attachViewHierarchy = true }
+        saveViewHierarchy("")
+        XCTAssertTrue(fixture.viewHierarchy.saveFilePathUsed?.hasSuffix("view-hierarchy.json") ?? false)
```

Comment:
`m`: Instead of asserting that the `saveFilePathUsed` has a suffix, I would prefer to validate that the view hierarchy gets stored to the path we passed in. So basically I would turn this into an integration test.

Reply:
We have such a test for SentryViewHierarchy, here I just want to know if `saveViewHierarchy` is calling the proper function from DependencyContainers`s viewHierarchy.

Reply:
But then this test should need some updates because it's not testing the logic properly, as we changed the code but not this test.